src/SignalR.AspNetCore/Akka.Streams.SignalR.AspNetCore/Akka.Streams.SignalR.AspNetCore.csproj

```diff
@@ -4,13 +4,13 @@
     <PropertyGroup>
         <AssemblyTitle>Akka.Streams.SignalR.AspNetCore</AssemblyTitle>
         <Description>SignalR ASP.Net Core connector for Akka.NET Streams</Description>
-        <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
         <PackageTags>$(AkkaPackageTags);signalr</PackageTags>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Akka.Streams" Version="$(AkkaVersion)" />
-        <PackageReference Include="microsoft.aspnetcore.signalr" Version="1.1.0" />
+        <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Need to reference the Microsoft.AspNetCore.App framework instead of referencing the SignalR nuget package directly

Reply:
the SignalR nuget package is bundled inside the framework

Reply:
LGTM